cfme/tests/distributed/test_appliance_replication.py

```diff
@@ -275,3 +278,41 @@ def test_replication_connect_to_vm_in_region(provider,
         view.login.click()
         view = vm.create_view(InfraVmDetailsView)
         wait_for(lambda: view.is_displayed, message="Wait for VM Details page")
+
+
+@pytest.mark.tier(2)
+@pytest.mark.ignore_stream("upstream")
+@pytest.mark.meta(automates=[1796681])
+def test_appliance_replicate_remote_down(temp_appliance_preconfig_funcscope_rhevm,
+       temp_appliance_unconfig_funcscope_rhevm):
+    """Test that the Replication tab displays in the global appliance UI when the remote appliance
+    database cannot be reached.
+
+    Bugzilla:
+        1796681
+
+    Metadata:
+        test_flag: replication
+
+    Polarion:
+        assignee: tpapaioa
+        initialEstimate: 1/4h
+        casecomponent: Appliance
+    """
+    remote_appliance = temp_appliance_preconfig_funcscope_rhevm
+    global_appliance = temp_appliance_unconfig_funcscope_rhevm
+
+    configure_replication_appliances(remote_appliance, global_appliance)
```

Comment:
Q: Should this be part of setup, i.e. in a fixture?

Reply:
Yes, see also https://github.com/ManageIQ/integration_tests/pull/9910#discussion_r383460315 . I'm planning to do that in a separate PR.